plugin/vscode/quick-lint-js/vscode/vscode-diag-reporter.h

```diff
@@ -144,7 +147,8 @@ class VSCode_Diag_Reporter final : public Diag_Reporter {
         /*env=*/this->env_,
         /*diagnostics=*/this->diagnostics_,
         /*locator=*/this->locator_,
-        /*document_uri=*/this->document_uri_);
+        /*document_uri=*/this->document_uri_,
+        /*qljs_message_translator=*/this->qljs_message_translator_);
```

Comment:
Nit: I prefer a shorter name:
```suggestion
        /*message_translator=*/this->message_translator_);
```